lib/metaschema-config/config.js

```diff
@@ -84,6 +87,29 @@ const preprocessor = schemas => {
   return arr;
 };
 
+const validate = (schema, path) => {
+  const errors = [];
+
+  for (const [prop, value] of Object.entries(schema)) {
+    const valueType = typeof value;
+    if (valueType === 'object') {
+      errors.push(...validate(value, `${path}.${prop}`, errors));
```

Comment:
The function only accepts 2 arguments, why do you pass `errors` as a third argument?